src/core/settings.h

```diff
@@ -89,6 +90,7 @@ typedef struct QUIC_SETTINGS_INTERNAL {
     uint16_t MaxBindingStatelessOperations;
     uint16_t StatelessOperationExpirationMs;
     uint16_t CongestionControlAlgorithm;
+    uint32_t IdleSrcCidChangeMs;
```

Comment:
For this internal struct, put this new `uint32_t` after all the other `uint32_t`s.